webapp/app/components/description/description.js

```diff
@@ -0,0 +1,101 @@
+import './description.less'
+
+import React, { useEffect, useState } from 'react'
+import * as R from 'ramda'
+import PropTypes from 'prop-types'
+
+import Icon from '@webapp/components/icon'
+import Tooltip from '@webapp/components/tooltip'
+
+import RichTextEditor from '@webapp/app/components/description/RichTextEditor'
+import useI18n from '@webapp/components/hooks/useI18n'
+import useCountryIso from '@webapp/components/hooks/useCountryIso'
+import useUserInfo from '@webapp/components/hooks/useUserInfo'
+import useGetRequest from '@webapp/components/hooks/useGetRequest'
+
+const getUrl = (countryIso, section, name) => `/api/country/descriptions/${countryIso}/${section}/${name}`
+
```

Comment:
Create a folder description where you have 
description
toggle,
title,
index.js (with export default Description)